aw_watcher_window/macos.py

```diff
@@ -1,20 +1,35 @@
 from ApplicationServices import AXIsProcessTrusted
-from AppKit import NSAlert
+from AppKit import NSAlert, NSAlertFirstButtonReturn, NSButton, NSWorkspace, NSURL
 import subprocess
 from subprocess import PIPE
 import os
+import time
 
 
 def getInfo() -> str:
-    accessibility_permissions = AXIsProcessTrusted()
-    #accessibility_permissions = False
+    #accessibility_permissions = AXIsProcessTrusted()
+    accessibility_permissions = False
     if not accessibility_permissions:
```

Comment:
I feel like this shouldn't be called in `getInfo()`, since if you miss the message it'll block all tracking until you close it. 

Instead, I'd run it in a separate thread upon start, before the main loop starts.

Reply:
Yeah fair point. I'll go for that implementation.

Reply:
resolved in f032ed1ad2a7903f42f3077d74a6d2c4f9b440f4